src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin: 0;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
add all modifiers here from the stars task to have the opportunity to change the nr of active stars by changing the modifier